src/integrations/GA4/browser.js

```diff
@@ -165,6 +154,9 @@ export default class GA4 {
     if (this.addSendToParameter) {
       params.send_to = this.measurementId;
     }
+    if (this.analytics.userId) {
+      params.user_id = this.analytics.userId;
```

Comment:
same here, please add a comment that says, what is achieved by adding user_id in params

Reply:
updated